usaspending_api/accounts/views/federal_accounts_v2.py

```diff
@@ -166,10 +193,101 @@ def post(self, request, pk, format=None):
         return Response(response)
 
 
+def filter_on(prefix, key, values):
+    if not isinstance(values, (list, tuple)):
+        values = [values, ]
+    return Q(**{'{}__{}__in'.format(prefix, key): values})
+
+
+def orred_filter_list(prefix, filters):
```

Comment:
I like these filter functions. It might be worthwhile to move them to a more centralized location for other views to use